apps/aevm/src/aevm_opcodes.erl

```diff
@@ -8,155 +8,154 @@
 
 -module(aevm_opcodes).
 
--export([ op_base_gas/1
-        , op_name/1
+-export([ op_base_gas/2
+        , op_name/2
         ]).
 
 -include_lib("aebytecode/include/aeb_opcodes.hrl").
--include("aevm_gas.hrl").
 %%====================================================================
 %% API
 %%====================================================================
 
-op_name(OP) -> element(1, opcode(OP)).
+op_name(OP, State) -> element(1, opcode(aevm_eeevm_state:gastable(State), OP)).
```

Comment:
The change `op_name/1` to `op_name/2` seems unnecessary. It can simply be forwarded to use `aeb_opcodes:mnemonic/1`.

Reply:
Addressed in 69f62ac9